src/Model/NoDriverGalleryManager.php

```diff
@@ -32,11 +32,26 @@ public function findAll(): array
         throw new NoDriverException();
     }
 
+    /**
+     * TODO: Add typehint with the drop of sonata-project/doctrine-extensions < 2.
+     *
+     * @param int|null $limit
+     * @param int|null $offset
```

Comment:
cant you add typehints since this is final and internal?

Reply:
No, because there is no typehint in doctrine-extensions 1.x.
I have to be compatible with both 1.x and 2.x